lightningd/notification.c

```diff
@@ -76,3 +77,17 @@ void notify_invoice_payment(struct lightningd *ld, struct amount_msat amount,
 	jsonrpc_notification_end(n);
 	plugins_notify(ld->plugins, take(n));
 }
+
+void notify_channel_opened(struct lightningd *ld, struct node_id *node_id,
+			 struct amount_sat *funding_sat, u32 *feerate_per_kw)
+{
+	struct jsonrpc_notification *n =
+	    jsonrpc_notification_start(NULL, notification_topics[4]);
```

Comment:
The use of indices here can lead to unwanted renamings if the `notification_topics` array get reordered. Prefer adding an inline string here.

Reply:
Probably appropriate to modify all of the notifications in a separate commit, then, as there is existing code which uses the same pattern.